src/UsgsAstroLsSensorModel.cpp

```diff
@@ -328,70 +331,69 @@ void UsgsAstroLsSensorModel::reset()
 
-  m_imageIdentifier = "";                    // 1
-  m_sensorType    = "USGSAstroLineScanner";  // 2
-  m_totalLines    = 0;                       // 3
-  m_totalSamples  = 0;                       // 4
-  m_offsetLines   = 0.0;                     // 7
-  m_offsetSamples = 0.0;                     // 8
-  m_platformFlag  = 1;                       // 9
-  m_aberrFlag     = 0;                       // 10
-  m_atmRefFlag    = 0;                       // 11
+  m_imageIdentifier = "";
+  m_sensorType    = "USGSAstroLineScanner";
+  m_totalLines    = 0;
+  m_totalSamples  = 0;
+  m_offsetLines   = 0.0;
+  m_offsetSamples = 0.0;
+  m_platformFlag  = 1;
+  m_aberrFlag     = 0;
+  m_atmRefFlag    = 0;
   m_intTimeLines.clear();
   m_intTimeStartTimes.clear();
   m_intTimes.clear();
-  m_startingEphemerisTime = 0.0;             // 13
-  m_centerEphemerisTime = 0.0;               // 14
-  m_detectorSampleSumming = 1.0;             // 15
-  m_startingSample = 1.0;                    // 16
-  m_ikCode = -85600;                         // 17
-  m_focal = 350.0;                           // 18
-  m_zDirection = 1.0;                        // 19
-  m_opticalDistCoef[0] = 0.0;                // 20
-  m_opticalDistCoef[1] = 0.0;                // 20
-  m_opticalDistCoef[2] = 0.0;                // 20
-  m_iTransS[0] = 0.0;                        // 21
-  m_iTransS[1] = 0.0;                        // 21
-  m_iTransS[2] = 150.0;                      // 21
-  m_iTransL[0] = 0.0;                        // 22
-  m_iTransL[1] = 150.0;                      // 22
-  m_iTransL[2] = 0.0;                        // 22
-  m_detectorSampleOrigin = 2500.0;           // 23
-  m_detectorLineOrigin = 0.0;                // 24
-  m_detectorLineOffset = 0.0;                // 25
-  m_semiMajorAxis = 3400000.0;               // 27
-  m_semiMinorAxis = 3350000.0;               // 28
-  m_referenceDateAndTime = "";               // 30
-  m_platformIdentifier = "";                 // 31
-  m_sensorIdentifier = "";                   // 32
-  m_trajectoryIdentifier = "";               // 33
-  m_collectionIdentifier = "";               // 33
-  m_refElevation = 30;                       // 34
-  m_minElevation = -8000.0;                  // 34
-  m_maxElevation =  8000.0;                  // 35
-  m_dtEphem = 2.0;                           // 36
-  m_t0Ephem = -70.0;                         // 37
-  m_dtQuat =  0.1;                           // 38
-  m_t0Quat = -40.0;                          // 39
-  m_numEphem = 0;                            // 40
-  m_numQuaternions = 0;                      // 41
-  m_ephemPts.clear();                        // 42
-  m_ephemRates.clear();                      // 43
-  m_quaternions.clear();                     // 44
+  m_startingEphemerisTime = 0.0;
+  m_centerEphemerisTime = 0.0;
+  m_detectorSampleSumming = 1.0;
+  m_startingSample = 1.0;
+  m_ikCode = -85600;
+  m_focal = 350.0;
+  m_zDirection = 1.0;
+  m_distortionType = (DistortionType)0;

```

Comment:
This should default to something from the enumeration, not 0 cast to the enumeration.